### DISCUSSION TITLE
react-pdf v6 + nextJS v13 integration not working with dynamic routes

### DISCUSSION THREAD

**Discussion by chrislicodes** [wojtekmaj/react-pdf#1388] [Q&A]
Hey, I forked a Sandbox and bumped the versions to the most recent ones: 

https://codesandbox.io/p/sandbox/react-pdf-next-js-forked-50d8vb?file=%2Fpages%2Findex.js

However, on the root index.js everything is working fine (<codeSandBoxUrl>, but as soon as I use dynamic route syntax <codeSandBoxUrl/test/1> the import breaks. I am not sure if this is a Next.js Bug or if I don't have the correct configuration setup.

Anyone else encoutered this issue?

**Comment by chrislicodes** [comment]
Nevermind, I found the Issue the second after posting this 😅

**Reply by vchettur** [reply]
@chrislicodes Can you tell me how you resolved this? I'm having the same exact issue. When I use App Router
<img width="1653" alt="Screen Shot 2023-11-24 at 7 53 18 AM" src="https://github.com/wojtekmaj/react-pdf/assets/121341192/a02d7042-1a39-4222-838d-fac978a2a807">
 without dynamic routes my react-pdf code works fine and loads pdfs. However, when I take the same code and use a dynamic route it fails. It says failed to load and **Unhandled Runtime Error
SyntaxError: expected expression, got '<'**
